src/ccm/ng-ce-ui/templates/virtualservice.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.global.istio.enabled }}
+{{- $fullName := include "ng-ce-ui.fullname" . -}}
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: ng-ce-ui
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  gateways:
+    {{- if .Values.global.istio.gateway.create }}
+    - istio-system/public
+    {{- else }}
+    {{- range .Values.global.istio.virtualService.gateways }}
+    - {{ . }}
+    {{- end }}
+    {{- end }}
+  hosts:
+    {{- range .Values.global.istio.virtualService.hosts }}
+    - {{ . }}
+    {{- end }}
+  http:
+  - name: ng-ce-ui
+    match:
+    - uri:
+        prefix: /ccmui
+    - uri:
+        prefix: /ccmui/
```

Comment:
We will try these rules on istio directly and check if the ccmui is getting hit or not. But theoritically looks good.

Reply:
Sure, thanks!